results/cmd/watcher/results.sql

```diff
@@ -0,0 +1,7 @@
+-- results_id serve as auto-generated key by sqltite to uniquely identify a taskrun log
```

Comment:
We shouldn't copy the database schema. We should reuse the existing one to ensure that the schemas stay consistent with each other.

Reply:
done